metron-interface/metron-alerts/src/app/alerts/alerts-list/alerts-list.component.ts

```diff
@@ -477,12 +499,17 @@ export class AlertsListComponent implements OnInit, OnDestroy {
     this.router.navigateByUrl('/alerts-list(dialog:save-search)');
   }
 
-  tryStartPolling() {
-    if (!this.isRefreshPaused) {
+  tryStartPolling(manualSearch?: SearchRequest) {
+    if (!this.isRefreshPaused && !manualSearch) {
       this.tryStopPolling();
       this.refreshTimer = this.searchService.pollSearch(this.queryBuilder.searchRequest).subscribe(results => {
         this.setData(results);
       });
+    } else if (!this.isRefreshPaused && manualSearch) {
```

Comment:
If polling and the manual query is enabled, we should be using the query provided in the input by the user.